test/integration/coronavirus_local_restrictions_test.rb

```diff
@@ -289,15 +289,15 @@ def then_i_see_details_of_national_restrictions
   end
 
   def then_i_see_the_results_for_wales
-    assert page.has_text?("Welsh Government")
```

Comment:
Is there a reason we shouldn't use the I18n here? Figure it would avoid having to update this in the future

Reply:
@JonathanHallam That's a good point. I've updated the test and fixed up.

Reply:
nice one, thanks :)